apps/extension/package.json

```diff
@@ -32,6 +32,7 @@
     "exponential-backoff": "^3.1.1",
     "framer-motion": "^11.0.22",
     "immer": "^10.0.4",
+    "lodash": "^4.17.21",
```

Comment:
tree shaking still sucks, and lodash is about about 1.5mb. i went through some effort to remove it as a dep a while back - if we're just going to use `debounce`, i would prefer to just write a debounce fn